projects.py

```diff
@@ -24,85 +24,69 @@ def parse_args():
     parser.add_argument(
         "--verbose", default=False, action="store_true", help="Show debugging output"
     )
+    parser.add_argument(
+        "--project", default=None, help="Run specific project only"
+    )
     return parser.parse_args()
 
-def run_projects(c):
+def run_projects(c, project_name):
     if c.household_match:
         log.debug("Processing households")
-        with open(Path(c.household_schema)) as schema_file:
-            household_schema = schema_file.read()
-            project_name = "fn-phone-addr-zip"
-            household_project = Project(
-                project_name,
-                household_schema,
-                c.systems,
-                c.entity_service_url,
-                c.blocked,
-            )
-            household_project.start_project()
-            for system in c.systems:
-                household_project.upload_clks(
-                    system, c.get_household_clks_raw(system, project_name)
-                )
-            household_project.start_run(c.matching_threshold)
-            running = True
-            print("\n--- RUNNING ---\n")
-            while running:
-                status = household_project.get_run_status()
-                print(status)
-                if status.get("state") == "completed":
-                    running = False
-                    break
-                time.sleep(SLEEP_TIME)
-            print("\n--- Getting results ---\n")
-            result_json = household_project.get_results()
-            Path(c.project_results_dir).mkdir(parents=True, exist_ok=True)
-            with open(Path(c.project_results_dir) / f'{project_name}.json', 'w') as json_file:
-                json.dump(result_json, json_file)
+        project_name = "fn-phone-addr-zip"
+        run_project(c, project_name, households=True)
     else:
         log.debug("Processing individuals")
         if c.blocked:
             log.debug("Blocked, extracting CLKs and blocks")
             for system in c.systems:
                 c.extract_clks(system)
                 c.extract_blocks(system)
+        if project_name:
+            run_project(c, project_name)
+        else:
+            for project_name in c.load_schema().keys():
+                run_project(c, project_name)
 
-        iter_num = 0
-        for project_name, schema in c.load_schema().items():
-            iter_num = iter_num + 1
-            project = Project(
-                project_name, schema, c.systems, c.entity_service_url, c.blocked
+def run_project(c, project_name=None, households=False):
+    schema = c.load_schema()[project_name]
```

Comment:
I don't think this will work correctly for households since the household schema comes from a different folder in data-owner-tools -- you'll need to read from the config.household_schema property, ex: https://github.com/mitre/linkage-agent-tools/blob/master/match.py#L39
(This could also be a new function in config.py to align with config.load_schema as used here, or a modification to load_schema)